getting-started/scaling-out.md

```diff
@@ -0,0 +1,178 @@
+Hypertables are used to handle large amount of data by breaking it up
+into smaller pieces (chunks), allowing operations to execute efficiently. When
+the amount of data is expected to be beyond what a single machine can handle,
+you can distribute the data chunks over several machines by using
+*distributed hypertables*.
+Distributed hypertables are similar to normal hypertables, but they
+add an additional layer of hypertable partitioning by distributing chunks
+across *data nodes*.
+
+Data nodes together with an *access node* constitute a multi-node cluster
+([architecture][]). All the nodes are TimescaleDB instances,
+i.e., hosts with a running PostgreSQL database and loaded TimescaleDB extension.
+While data nodes store distributed chunks, the access node is
+the entry point for clients to access distributed hypertables.
+
+## Working with Data Nodes
+
+Data nodes act as the containers for the hypertable chunks and are
+necessary to create distributed hypertables. Data nodes are
+added to the current database on an access node
+using [`add_data_node`][add_data_node]
+and removed using [`delete_data_node`][delete_data_node].
+
+Note that:
+
+* Data nodes are added as objects locally and connect to an
+  existing PostgreSQL instance.
+
+* You should already have a running PostgreSQL server on the data node host.
+
+When creating the data node, you should:
+
+* Provide a name to use when referring to the data node from
+  this database.
+
+* Provide the host where the hypertable partition for the distributed
+  hypertable should be stored.
+  
+* Provide password, which will be used during access to
+  the created remote data node.
+
+* Provide bootstrap user and password, which is used to
+  create the data node. If the current user can be used, then 
+  the boostrap user and password can be omitted.
+  
+* Ensure that the data node has password authentication enabled 
+  in their `pg_hba.conf` files for any non-superusers.
+
+* Ensure that the bootstrap user used for connecting to the data node has
+  `CREATEDB` privilege or is a superuser. 
+  This is necessary since
+  `add_data_node` expects to be able to create a
+  database on the remote data node and create
+  TimescaleDB extension in it.
+  
+* Ensure that the local user has `USAGE` privilege on the `timescaledb_fdw` 
+  foreign data wrapper on the access node.
+  
+```sql
+SELECT add_data_node('node1', host => 'dn1.example.com',
+  password=>'<remote_password>', bootstrap_user=>'<superuser>',
+  bootstrap_password=>'<superuser_password>');
+SELECT add_data_node('node2', host => 'dn2.example.com',
+  password=>'<remote_password>', bootstrap_user=>'<superuser>',
+  bootstrap_password=>'<superuser_password>');
+```
+
+Any additional users that will access a distributed hypertable currently 
+need their own user mappings per data node with a `user` and `password` option. 
+A user mapping can be created for a data node as follows: 
+
+```sql
+CREATE USER MAPPING FOR foo SERVER node1 
```

Comment:
I'd go with <user> here as well in place of `foo`

Reply:
Thank you for noticing it. Done.